guides/common/modules/proc_installing-the-kernelcare-package-on-managed-hosts.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Installing_the_KernelCare_Package_on_Managed_Hosts_{context}"]
+= Installing the KernelCare Package on Managed Hosts
+
+You can use `kernelcare` to patch the Linux kernel on managed hosts without rebooting them.
+
+.Prerequisite
```

Comment:
I suggest using plural even if it is just one prerequisite. The structure of procedural modules is than more unified. It is not anything that matters too much though, so I won't object if you leave it as-is.

Reply:
We currently use singular for this case. If you want to make an argument to switching over to plural everywhere, I'd be fine with this & we could probably change this using `sed`.